src/api/app/views/webui/users/tokens/index.html.haml

```diff
@@ -0,0 +1,60 @@
+- @pagetitle = 'Tokens'
+
+.card
+  .card-body
+    %h3= @pagetitle
+
+    .text-right
+      = link_to new_token_path do
+        %i.fas.fa-plus-circle.text-primary
+        Create Token
```

Comment:
Comparing with other UIs, we have `Create (...)` link at the bottom left of lists. Examples:
- Create Package: https://obs-reviewlab.opensuse.org/eduardoj-webui_for_user_tokens/project/show/home:Admin
- Add Member: https://obs-reviewlab.opensuse.org/eduardoj-webui_for_user_tokens/groups/group_1

Reply:
Ok. I will move it to the bottom left of the list.

Reply:
Done.